R/FilterState.R

```diff
@@ -3,17 +3,16 @@
 #' @name FilterState
 #' @docType class
 #'
-#' @title `FilterState` abstract Class
+#' @title `FilterState` abstract class
 #'
 #' @description Abstract class to encapsulate single filter state.
 #'
 #' @details
-#' This class is responsible for managing single filter item within
-#' `FilteredData` class object. Filter states depend on the variable type:
+#' This class is responsible for managing a single filter item within a `FilteredData` object
+#' and outputs a condition call (logical predicate) for subsetting one variable.
+#' Filter states depend on the variable type:
 #' (`logical`, `integer`, `numeric`, `factor`, `character`, `Date`, `POSIXct`, `POSIXlt`)
```

Comment:
(for some reason I cannot highlight appropriate area - sorry)
Would you mind reorder bullet lists (right below) to be in line with the order of elements enumerated here.